model/test_results_storage.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import "github.com/mongodb/anser/bsonutil"
+
+// TestResultsArtifactInfo describes a bucket of test results for a given task
+// execution stored in some kind of offline storage. It is the bridge between
+// pail-backed offline log storage and the cedar-based log metadata storage.
+// The prefix field indicates the name of the "sub-bucket". The top level
+// bucket is accesible via the cedar.Environment interface.
+type TestResultsArtifactInfo struct {
+	Type    PailType        `bson:"type"`
+	Prefix  string          `bson:"prefix"`
+	Version int             `bson:"version"`
+	Tests   map[string]bool `bson:"tests,omitempty"`
```

Comment:
what's the bool? I think a []string{} is better, but I still don't know that this makes sense in the DB

Reply:
The bool is to track if it has been accounted for in the historical stats data, in case we have to do a backfill